src/Console/Command/FixCommand.php

```diff
@@ -151,6 +151,11 @@ public function getHelp(): string
 
                 <info>$ php %command.full_name% /path/to/project --rules='{"concat_space": {"spacing": "none"}}'</info>
 
+            The <comment>--rules-base64</comment> option allows rules to be specified as a base64 encoded string. This can be useful for
```

Comment:
before coming to solution, would you mind to provide a test where we can clearly reproduce (on CI, i must share I have no Windows machine available) the original issue ?
(i guess it would be more like Integration or Smoke test, rather than unit test)